src/transactionIntentsFactories/delegationTransactionIntentsFactory.spec.ts

```diff
@@ -0,0 +1,301 @@
+import BigNumber from "bignumber.js";
+import { Address } from "../address";
+import { DelegationTransactionIntentsFactory } from "./delegationTransactionIntentsFactory";
+import { assert } from "chai";
+import { DELEGATION_MANAGER_SC_ADDRESS } from "../constants";
+import { ValidatorPublicKey } from "@multiversx/sdk-wallet-next";
+import { SignableMessage } from "../signableMessage";
+
+describe("test delegation intents factory", function () {
+    const delegationFactory = new DelegationTransactionIntentsFactory({
```

Comment:
If we add a config like here:

https://github.com/multiversx/mx-sdk-py-core/blob/main/multiversx_sdk_core/transaction_intents_factories/transaction_intents_factory_config.py

Then we can also use it in the unit tests (easier).

Reply:
Created the config file.